flytekit/clis/sdk_in_container/register.py

```diff
@@ -13,21 +16,31 @@
 def register_all(project, domain, pkgs, test, version):
     if test:
         click.echo('Test switch enabled, not doing anything...')
-
     click.echo('Running task, workflow, and launch plan registration for {}, {}, {} with version {}'.format(
         project, domain, pkgs, version))
 
     # m = module (i.e. python file)
     # k = value of dir(m), type str
     # o = object (e.g. SdkWorkflow)
+    loaded_entities = []
     for m, k, o in iterate_registerable_entities_in_order(pkgs):
         name = _utils.fqdn(m.__name__, k, entity_type=o.resource_type)
-
+        _logging.debug("Found module {}\n   K: {} Instantiated in {}".format(m, k, o._instantiated_in))
```

Comment:
so the default logging is to the console, would this be weird to look at?
How about you enable click.echo when debug is on?

Reply:
what I mean is click.debug_echo?

Reply:
Normally this wouldn't print at all actually, it'll only show up if you set your logging level elsewhere to debug, which I do when I'm working on this.  But I think too many people have complained about this file, I think I'll just revert everything here.